tests/integration_tests/test_kani.py

```diff
@@ -34,7 +34,7 @@ def test_kani(results_dir):
     # --output-format terse is required by -j
     # -Z unstable-options is needed to enable the other `-Z` flags
     _, stdout, _ = utils.check_output(
-        "cargo kani -Z unstable-options -Z stubbing -Z function-contracts -Z restrict-vtable -j --output-format terse --harness-timeout 30m",
+        "cargo kani -Z unstable-options -Z stubbing -Z function-contracts -Z restrict-vtable -j --output-format terse --harness-timeout 40m",
```

Comment:
is this bumped to 40m from the previous commit on purpose?

Reply:
no, that's because I forgot this PR had multiple commits this morning when I tried to amend. fixed, thanks